Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Cloud_IAM_User_Access_Investigation_README.md

```diff
@@ -0,0 +1,64 @@
+Investigate and respond to Cortex XDR Cloud alerts where a Cloud IAM user`s access key is used suspiciously to access the cloud environment. 
+The following alerts are supported for AWS, Azure, and GCP environments.
+- Penetration testing tool attempt
+- Penetration testing tool activity
+- Suspicious API call from a Tor exit node
+
+
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+* Cloud Response - Generic
+* Account Enrichment - Generic v2.1
+* Cloud IAM Enrichment - Generic
+
+### Integrations
+
+* XDR_iocs
+* XQLQueryingEngine
```

Comment:
The playbook doesnt use the XQL integration. am I wrong? (also the IOC one)

Reply:
Done